dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -136,7 +136,8 @@ public void testComponent_success() throws IOException, ValidityException, Parsi
       Nodes greens = document.query("//h3[@style='color: green']");
       Assert.assertTrue(greens.size() >= 2);
       Nodes pres = document.query("//pre");
-      Assert.assertEquals(0, pres.size());
+      // There's a pre tag for dependency
+      Assert.assertEquals(1, pres.size());
```

Comment:
This could fail if there's another pre element for the wrong reason. I wasn't planning on one, but now there is. :-) Maybe add an id attribute and check for that to distinguish? Or check the contents of the element?

Reply:
Enhanced test cases to distinguish pre tags.